docs/sources/plugins/catalog.md

```diff
@@ -6,17 +6,56 @@ weight = 1
 
 # Plugin catalog
 
-Catalog for Grafana is a plugin bundled with Grafana versions 8.0+. It allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the Marketplace.
+The Plugin catalog, new in Grafana 8.0, allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the catalog.
 
-[screenshot placeholder]
+<div class="medium-6 columns">
+  <video width="800" height="500" controls>
+    <source src="/assets/videos/plugins-catalog-install-8-0.mp4" type="video/mp4">
+    Your browser does not support the video tag.
+  </video>
+</div>
 
-To use the Catalog for Grafana, you first need to enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#catalog_app_enabled" >}}).
+Before you can use the Plugin catalog, you must enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#plugin_admin_enabled" >}}) file.
+Before following the steps below, make sure you are logged in as a Grafana administrator.
 
-## Install a plugin from the Catalog
-To install a plugin ...
+## Browse plugins
 
-### Updating a plugin
-To update a plugin ...
+1. In Grafana, navigate to **Configuration > Plugins**.
+1. Click the **Install &amp; manage plugins** button.
+1. You can now browse and search for plugins.
+1. You can also browse existing plugins by navigating to the **Library** tab.
 
-## Uninstall a plugin from the Catalog
-To uninstall a plugin ...
+![Plugin catalog browse](/img/docs/plugins/plugins-catalog-browse-8-0.png)
+
+## Install a plugin
+
+1. In Grafana, navigate to **Configuration > Plugins**.
+1. Click the **Install &amp; manage plugins** button.
```

Comment:
Do not use "button." Just "Click **Install & manage plugins"**."

Though really, you could just rename the button **Manage plugins** and be fine.